04- Views and URLs/README.md

```diff
@@ -0,0 +1,790 @@
+# ویوها و URLها
+
+در این فصل ما در مورد مباحث زیر بحث خواهیم کرد:
+
+- ویوهای مبتنی بر کلاس و مبتنی بر تابع
+- میکسین‌ها
+- دکوراتورها
+- الگوهای مرسوم ویو
+- طراحی کردن URLها
+- کار کردن با ری‌اکت و دیگر فرانت‌اندهای جاوا اسکریپت
+
+## نگاهی به ویو از بالا 
+
+در جنگو، ویوها به عنوان فراخوانی کننده تعریف می‌شوند که درخواست‌ها را می‌پذیرند و پاسخ‌ها را برمی‌گردانند. ویوها معمولاً یک تابع یا کلاسی به همراه متود کلاسی مخصوص مثل ```()as_view``` هستند.
+
+در هر دو مورد ما یک تابع پایتون معمولی میسازیم که ```HTTPRequest``` را به عنوان آرگومان اول می‌گیرد و ```HTTPResponse``` را به عنوان پاسخ برمی‌گرداند.
+یک ```پیکربندی URL یا (URLConf)``` نیز میتواند به عنوان آرگومان اضافه به این تابع فرستاده شود. این آرگومان‌ها می‌توانند از، بخشی از URL گرفته شوند و یا مقدار آن به صورت پیشفرض معین شده باشد.
+
+نمونه یک ویوی ساده به شکل زیر است:
+
```

Comment:
Please also write the name of the language above the codes. like [this](https://www.freecodecamp.org/news/how-to-format-code-in-markdown/#:~:text=There%20are%20two%20ways%20to,will%20apply%20syntax%20highlighting%20to.):

Because the keywords of the programming language are displayed in color. Thank.